### PR TITLE
Case merge

### PR DESCRIPTION
**Pull request by mew2057**
This basically clears cloudpak cert for the operator.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,14 @@
 services: docker
 sudo: required
 dist: xenial
-
+#group: bluezone
```
Comment (whowutwut) [inline .travis.yml]: delete this?  seems like it's being used in the .travis-pri.yaml file

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+###############################################################################
+# Licensed Materials - Property of IBM.
+# Copyright IBM Corporation 2017. All Rights Reserved.
```
Comment (whowutwut) [inline Makefile]: Nitpicking, should we update to 2020?

File: stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/LICENSE [hunk 3]
```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 IBM Corporation
+   Copyright [yyyy] [name of copyright owner]
```
Comment (whowutwut) [inline stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/LICENSE]: Seems like we blanked this out...  does it get auto filled in?

File: stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/deploy/role_binding.yaml [hunk 4]
```diff
@@ -6,6 +6,7 @@ metadata:
     app.kubernetes.io/managed-by: ibm-spectrum-scale-csi-operator
     app.kubernetes.io/name: ibm-spectrum-scale-csi-operator
     product: ibm-spectrum-scale-csi
+    release: ibm-spectrum-scale-csi-operator
```
Comment (whowutwut) [inline stable/ibm-spectrum-scale-csi-operator-bundle/operators/ibm-spectrum-scale-csi-operator/deploy/role_binding.yaml]: what happens here when we merge the repos?  product vs release names?

File: stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/LICENSE (hunk 3 above)
Comment (mew2057) [reply stable/ibm-spectrum-scale-csi-operator-bundle/case/ibm-spectrum-scale-csi-operator/LICENSE]: We're gonna need to  fix that whoops

**Pull request merged**